Client/public/index.html

```diff
@@ -46,13 +46,60 @@
         <table id="table1" cellpadding="0" cellspacing="0" border="0" >
           <tr> 
             <td> <a href="https://github.com/Pyxsys/SOEN390_19"> <img  alt="logo" src="https://user-images.githubusercontent.com/58408904/113503733-a1fab480-9501-11eb-8147-4f6d9b1a2a97.png" width="70" height="70"> </a>  <figcaption> Github</figcaption> </td>
-            <td> <img src="https://user-images.githubusercontent.com/58408904/113531632-43801580-9597-11eb-959f-ace57c415f69.png" alt="logo" width="70" height="70"  /> <figcaption> Facebook</figcaption></td>
-            <td> <img src="https://user-images.githubusercontent.com/58408904/113532020-5e06be80-9598-11eb-8226-a9331a840a93.png" alt="logo" width="70" height="70"  /> <figcaption> Instagram</figcaption></td>
-            <td> <img src="https://user-images.githubusercontent.com/58408904/113531959-3152a700-9598-11eb-82ea-ddb7a20ec55e.png" alt="logo" width="80" height="70"  /> <figcaption> Twitter</figcaption></td>
-            <td> <img src="https://user-images.githubusercontent.com/58408904/113503728-9dce9700-9501-11eb-90f5-bb6f63fde848.png" alt="logo" width="70" height="70"  /> <figcaption> About Us</figcaption> </td>
+            <td> <a href="https://www.facebook.com/"> <img src="https://user-images.githubusercontent.com/58408904/113531632-43801580-9597-11eb-959f-ace57c415f69.png" alt="logo" width="70" height="70"  /> </a>  <figcaption> Facebook</figcaption></td>
+            <td> <a href="https://www.instagram.com/"> <img src="https://user-images.githubusercontent.com/58408904/113532020-5e06be80-9598-11eb-8226-a9331a840a93.png" alt="logo" width="70" height="70"  /> </a>  <figcaption> Instagram</figcaption></td>
+            <td> <a href="https://twitter.com/?lang=en"> <img src="https://user-images.githubusercontent.com/58408904/113531959-3152a700-9598-11eb-82ea-ddb7a20ec55e.png" alt="logo" width="80" height="70"  /> </a>  <figcaption> Twitter</figcaption></td>
+            <td> <a href="https://github.com/Pyxsys/SOEN390_19#readme"> <img src="https://user-images.githubusercontent.com/58408904/113503728-9dce9700-9501-11eb-90f5-bb6f63fde848.png" alt="logo" width="70" height="70"  /> </a>  <figcaption> About Us</figcaption> </td>
           </tr> 
        </table>
     </footer>
     </div>
-  
+   
+      </head>
+      <body>
+      <button id="InstButton">Instructions</button>
```

Comment:
Element is visible from everywhere on the site but is only concerned with a single page.

Reply:
but it needs to be everywhere .This is what me and nathan had in mind. To show how to access and know how to use website no matter where you are

Reply:
and it looks really nice in the home page on the top right